### PR TITLE
Scaffold header

### PR DESCRIPTION
**Pull request by mikeyrayvon**
the grid is business. this was so smooth

### CODE REVIEW

File: css/responsive/mobile.styl [hunk 1]
```diff
@@ -0,0 +1,59 @@
+.site-title
+  text-align: center
+  &.color-white
```
Comment (varffs) [inline css/responsive/mobile.styl]: i think better for these kind of active states on a overlay or whatever to add a body class with js and style from there. more readable and can be used if needed later down the line

File: js/main.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ Site = {
     });
 
     $(document).ready(function () {
-
+      _this.toggleMenu();
```
Comment (varffs) [inline js/main.js]: yh Menu its own object

File: header.php [hunk 3]
```diff
@@ -27,6 +27,46 @@
...
-    <h1><a href="<?php echo home_url(); ?>"><?php bloginfo('name'); ?></a></h1>
+  <header id="header" class="padding-top-small padding-bottom-small">
+    <h1 class="u-visuallyhidden"><?php bloginfo('name'); ?></h1>
+
+    <div class="container">
+      <div class="grid-row">
+        <div class="grid-item item-s-12 item-m-6">
+          <div id="mobile-header" class="site-title">
+            <a href="<?php echo home_url(); ?>">LANZA</a>
+            <div class="mobile-toggle mobile-only">=</div>
```
Comment (varffs) [inline header.php]: if there is only one mobile-toggle it can have an id

File: header.php [hunk 4]
```diff
@@ -27,6 +27,46 @@
...
+          </div>
+        </div>
+        <div class="grid-item item-s-12 item-m-6 grid-column no-gutter" id="nav-holder">
```
Comment (varffs) [inline header.php:45]: why do you remove gutter on the parent not the children? won't that confuse the columns. As I see on my local there is no left padding on this section but the gutter for the language toggle puts the proper right padding

Comment (varffs) [inline header.php:45]: also why subgrid like this? it could just be title m-6 links m-4 language toggle m-2

File: header.php [hunk 5]
```diff
@@ -27,6 +27,46 @@
...
 
-  <header id="header">
-    <h1><a href="<?php echo home_url(); ?>"><?php bloginfo('name'); ?></a></h1>
+  <header id="header" class="padding-top-small padding-bottom-small">
+    <h1 class="u-visuallyhidden"><?php bloginfo('name'); ?></h1>
+
+    <div class="container">
+      <div class="grid-row">
+        <div class="grid-item item-s-12 item-m-6">
+          <div id="mobile-header" class="site-title">
```
Comment (varffs) [inline header.php:36]: this isn't just the mobile header but the title aways so the id is confusing

**Comment by varffs** [comment]
imo might as well write different markup for mobile and desktop when the header/menu functions so differently. It adds very little extra weight to the site and the division makes things a bit clearer

File: css/responsive/mobile.styl (hunk 1 above)
Comment (mikeyrayvon) [reply css/responsive/mobile.styl]: ok yea good call

File: header.php (hunk 4 above)
Comment (mikeyrayvon) [reply header.php:45]: because i wanted nav-holder so i could fit those elements properly in the mobile menu

Comment (mikeyrayvon) [reply header.php:45]: then i took the padding off the nav holder, and kept the padding on its children, except removing left padding from the nav elem so i could center its left

**Comment by mikeyrayvon** [comment]
maybe i should add the toggle icons that i have from EF? it looks like theyre the same assets.

**Comment by varffs** [comment]
screen shot 2016-09-07 at 15 35 33

removing the padding on `#nav-menu` breaks the grid columns

**Comment by mikeyrayvon** [comment]
ok fixed that

**Comment by mikeyrayvon** [comment]
it doesnt make it clearer for me. i prefer keeping the markup consistent. isnt it better semantically to have 1 nav elem instead of repeating markup?

**Comment by mikeyrayvon** [comment]
i made it work so...

**Comment by varffs** [comment]
yes it works and it is better semantically I think (tho actually no idea how display values and media queries effect screen readers?)

I just like the code separated for stuff like this. There are often cases cus as Gonzalo L where using the same markup would be near impossible, so I prefer splitting as a pattern. But merging this

**Comment by varffs** [comment]
Also the hamburger is weird here as well as EF

**Comment by mikeyrayvon** [comment]
idk what to do about the hamburger rendering. its their svg

**Pull request merged**